src/components/RegistrationDetailsPage.js

```diff
@@ -154,8 +155,42 @@ const RegistrationAnswers = ({ questions }) => {
   )
 }
 
+const GroupDetails = ({ groupDetails }) => {
+  if (!groupDetails) {
+    return (
+      <div>
+        <h2>This user is currently not part of any group.</h2>
```

Comment:
Would a more user-friendly copy be "You are currently not part of any group"?

Reply:
How about not showing group details at all if user is not in group?